src/OutlierRemoval/IQROutlierRemoval.cs

```diff
@@ -0,0 +1,42 @@
+﻿using AiDotNet.Quartile;
+
+namespace AiDotNet.OutlierRemoval;
+
+public class IQROutlierRemoval : IOutlierRemoval
+{
+    public IQROutlierRemoval(IQuartile? quartile = null)
+    {
+        Quartile = quartile;
+    }
+
+    internal int[] DetermineIQR(double[] unfiltered)
+    {
+        //initializes new type StandardQuartile of class IQuartile only if type is null else leaves the same.
+        var (q1Value, _, q3Value) = QuartileHelper.FindQuartiles(unfiltered, Quartile ?? new StandardQuartile());
+        var iQR = q3Value - q1Value;
+        var minLimit = q1Value - (1.5 * iQR);
```

Comment:
You should set 1.5 * iQR to a new variable and reference this variable in both spots